docs/metastore_upgrade_guide.md

```diff
@@ -0,0 +1,224 @@
+## Table Upgrade Guide
+This document provides a guide for upgrading the Hive metastore objects to UC using UCX.
+The metastore upgrade process is composed of multiple steps.
+To effectively upgrade the metastores, four principal operations are required:
+1. Assess - In this step, you evaluate the existing HMS tables identified for upgrade to determine the right approach for upgrade. This step is a prerequisite and is performed by the assessment workflow.
+2. Create - In this step, you create the required UC assets such as, Metastore, Catalog, Schema, Storage Credentials, External Locations. This step is part of the upgrade process.
+3. Upgrade/Grant these are two steps that UCX combine.
+   4. Upgrade - The metastores objects (tables/views) will be converted to a format supported by UC 
+   4. Grant - The table upgrade the newly created object the same permission as the original object.
+
+## Prerequisites
+For UCX to be able to upgrade the metastore. The following prerequisites must be met:
+1. UCX must be installed and configured on the workspace. For more information on how to install UCX, refer to the [external_hms_glue.md](external_hms_glue.md).
+2. In case of an external metastore (such as GLUE), UCX has to be configured to attach to the metastore. For more information on how to configure UCX to attach to an external metastore, refer to the [External Metastore Guide]().
+3. The assessment workflow must be run.
+4. It is recommended that the group migration process will be completed before upgrading the metastore. For more information on how to migrate groups, refer to the [UCX Readme](../README.md).
```

Comment:
make link to a header, not the whole readme

Reply:
Fixed